Tests/TestingApps/PurchaseTesterSwiftUI/Core/ConfiguredPurchases.swift

```diff
@@ -38,6 +38,7 @@ public final class ConfiguredPurchases {
             with: .builder(withAPIKey: apiKey)
                 .with(usesStoreKit2IfAvailable: useStoreKit2)
                 .with(observerMode: observerMode)
+                .with(dangerousSettings: DangerousSettings(customEntitlementComputation: true))
```

Comment:
This could be a UI setting :)

Reply:
I was thinking that too, I'll make a PR for it after I clean this up

Reply:
turns out a UI setting wouldn't really work here, for 2 reasons: 

1. the appUserID **must** be set if this mode is enabled. Totally fixable, though, and NaBD, we'd just add a field for it
2. and more important: PurchaseTester is really not built around this mode. So it'll still call customerInfo and other APIs, so it's not representative of this behavior in the slightest. 

I was thinking of just adding another TestingApp, single page, that just offers a button to initiate a purchase and maybe prints out customerInfo stream events when they come in, as well as offerings. Very simple. I already had to write it to test this out in the first place. 

What do you think?

Reply:
> turns out a UI setting wouldn't really work here, for 2 reasons:

Oh yeah good point. Do you think it's worth maintaining a separate app though?